src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -264,7 +272,22 @@
         limit-insufficient?                         (> (controlled-input/numeric-value input-state)
                                                        current-limit)
         should-try-again?                           (and (not limit-insufficient?) no-routes-found?)
-        current-address                             (rf/sub [:wallet/current-viewing-account-address])]
+        current-address                             (rf/sub [:wallet/current-viewing-account-address])
+        owned-eth-token                             (rf/sub [:wallet/token-by-symbol
+                                                             "ETH"
+                                                             enabled-from-chain-ids])
+        not-enough-asset?                           (and
+                                                     (or no-routes-found? limit-insufficient?)
+                                                     (not-empty sender-network-values)
+                                                     (if (= token-symbol "ETH")
```

Comment:
Better if we can avoid token symbols floating around or magical values in general. Could be a constant, but then it's also important to reuse the same constant throughout the codebase, otherwise it's not much better.

Reply:
Done